clar2wasm/Cargo.toml

```diff
@@ -24,6 +24,12 @@ wat = "1.0.74"
 flamegraph = []
 pb = []
 
+# Test-specific features
+test-clarity-v1 = []
+test-clarity-v2 = []
+test-clarity-v3 = []
+test-clarity-latest = ["test-clarity-v3"]
```

Comment:
Why do we need `test-clarity-latest`? vs defaulting to the latest version if no other feature was set

Reply:
Removed `test-clarity-latest`. Added a default feature `test-clarity-v3`.